gym-unity/README.md

```diff
@@ -67,6 +67,7 @@ The returned environment `env` will function as a gym.
   be accessed from the `info` provided by `env.step(action)`.
 - Stacked vector observations are not supported.
 - Environment registration for use with `gym.make()` is currently not supported.
+- Display rendering is not supported. Calling `env.render()` might not work as expected in display.
```

Comment:
It is not true that it is not supported in my opinion. I think we could :
 - remove the `env.render()` method and then just say "Display rendering is not supported."
 - keep `env.render` as is and say here : "Calling `env.render()` will return the latest visual observation. It will not render a new frame of the environment." (also change the docstring on the method.)
 - implement a proper `env.render()`. This is a lot of work for not a lot of benefit IMO.

Reply:
For simplicity I prefer not to change/remove the interface now. So I went with second option and updated the doc, let me know what you think.